hazelcast-client/src/main/java/com/hazelcast/client/spi/ClientContext.java

```diff
@@ -84,6 +87,10 @@ public NearCacheManager getNearCacheManager() {
         return nearCacheManager;
     }
 
+    public ClientLoggingService getClientLoggingService() {
```

Comment:
I would rename this to 'getLoggingService' and return a LoggingService instance.